src/server/logs/cmds/cmds_test.go

```diff
@@ -468,3 +468,38 @@ func TestGetLogs_from_to(t *testing.T) {
 		"To", to.Format(time.RFC3339Nano),
 	).Run())
 }
+
+func TestGetLogs_limit(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	var (
+		ctx          = pctx.TestContext(t)
+		buildEntries = func() []loki.Entry {
+			var entries []loki.Entry
+			for i := -99; i <= 0; i++ {
+				entries = append(entries, loki.Entry{
+					Timestamp: time.Now().Add(time.Duration(i) * time.Second),
+					Line:      fmt.Sprintf("%v foo", i),
+				})
+			}
+			return entries
+		}
+		env = realEnvWithLoki(ctx, t, buildEntries())
+		c   = env.PachClient
+	)
+	mockInspectCluster(env)
```

Comment:
I think is unnecessary copypasta.  realEnv has had an admin server for a while; not forever but it's there now.